src/components/about-us/opening/background.js

```diff
@@ -0,0 +1,201 @@
+    display: none;
+  `}
+`
+
+const Triangle = styled.div`
+  ${screen.desktop`
+    width: ${triangleSizes.desktop.width};
+    height: ${triangleSizes.desktop.height};
+    border-top: solid ${triangleSizes.desktop.height} #c7000a;
+    border-left: solid calc(${triangleSizes.desktop.width} * (3 / 8)) transparent;
+    border-right: solid calc(${triangleSizes.desktop.width} * (5 / 8)) transparent;  
+  `}
+  ${screen.overDesktop`
+    width: ${triangleSizes.hd.width};
+    height: ${triangleSizes.hd.height};
+    border-top: solid ${triangleSizes.hd.height} #c7000a;
+    border-left: solid calc(${triangleSizes.hd.width} * (3 / 8)) transparent;
+    border-right: solid calc(${triangleSizes.hd.width} * (5 / 8)) transparent;
+  `}  
+`
+
+const TriangleRightTop = Triangle.extend`
+  position: absolute;
+  right: 0;
+  top: 0;
+`
+
+const TriangleRightBottom = Triangle.extend`
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  transform: rotateX(180deg);  
+`
+
+const MobileTriangle = styled.div`
+  width: ${triangleSizes.mobile.width};
+  height: ${triangleSizes.mobile.height};
+  border-left: solid ${triangleSizes.mobile.width} #c7000a;
+  border-top: solid calc(${triangleSizes.mobile.height} * (3 / 8)) transparent;
+  border-bottom: solid calc(${triangleSizes.mobile.height} * (5 / 8)) transparent;
+  ${screen.tablet`
+    width: ${triangleSizes.tablet.width};
+    height: ${triangleSizes.tablet.height};
+    border-top: solid calc(${triangleSizes.tablet.height} * (3 / 8)) transparent;
+    border-bottom: solid calc(${triangleSizes.tablet.height} * (5 / 8)) transparent;
+  `}      
+`
+
+const TriangleRight = MobileTriangle.extend`
+  position: absolute;
+  right: 0;
+  top: calc(50% + ${triangleSizes.mobile.height} * (1 / 8));
+  transform: translateY(-50%) rotateY(180deg);
+  ${screen.tablet`
+    top: calc(50% + ${triangleSizes.tablet.height} * (1 / 8));
+  `}      
+`
+
+const TriangleLeft = MobileTriangle.extend`
+  position: absolute;
+  left: 0;
+  top: calc(50% + ${triangleSizes.mobile.height} * (1 / 8));
+  transform: translateY(-50%);
+  ${screen.tablet`
+    top: calc(50% + ${triangleSizes.tablet.height} * (1 / 8));
+  `}
+`
+
+const Divider = styled.div`
+  position: absolute;
+  right: 0;
+  height: 100%;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  border-left: solid 1px #c7000a;
+  width: calc(${triangleSizes.desktop.width} * (5 / 8));
+  ${screen.overDesktop`
+    width: calc(${triangleSizes.hd.width} * (5 / 8));
+  `}  
+  ${screen.tabletBelow`
+    top: 0;
+    width: 100%;
+    height: 50%;
+    border-left: none;
+    border-bottom: solid 1px #c7000a;
+  `}  
+`
+
+const Billboard = styled.div`
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-image: url(${billboard});
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: contain;
+  margin: 10% 5%;
+  width: 100%;
+  height: 100%;
+`
```

Comment:
Since `Billboard` and `Divider` are very different on mobile and desktop mockup,
I suggest you can split them to `MobileDivider` and `DesktopDivider`,  and use as simple CSS styles as you can. Currently, it's quite difficult to understand these (flex) styles.